watertap/unit_models/tests/unit_test_harness.py

```diff
@@ -59,8 +59,8 @@ def configure_class(self):
         self.default_zero = 1e-10
 
         # arguments for solver tolerance
-        self.default_absolute_tolerance = 1e-12
-        self.default_relative_tolerance = 1e-6
+        self.default_absolute_tolerance = 1e-5
+        self.default_relative_tolerance = 1e-3
```

Comment:
Same comment as https://github.com/watertap-org/watertap/pull/1367#discussion_r1579448115. Are these tolerances reasonable? If so, I think this and #1367 are good-to-go. If not, then we would need to address why some of the solutions fail at lower tolerances.